lib/resources/example_json/soarian_examples_document_reference.rb

```diff
@@ -0,0 +1,169 @@
+module Cerner
+  module Resources
+
+    SOARIAN_DOCUMENT_REFERENCE_DOCREF_BUNDLE ||= {
+	
+    "resourceType": "Bundle",
+    "id": "3def23df-44bf-4006-ab1c-72d0c176b1b0",
+    "meta": {
+        "lastUpdated": "2017-06-05T14:18:31.429-04:00"
+    },
+    "extension": [
+        {
+            "url": "http://cerner.hs.fhir.com/StructureDefinition/bundle-result-messages",
+            "valueExtension": {
+                "url": "http://cerner.hs.fhir.com/StructureDefinition/bundle-result-messages",
+                "extension": [
+                    {
+                        "url": "Severity",
+                        "valueString": "info"
+                    },
+                    {
+                        "url": "Code",
+                        "valueString": "papi022"
+                    },
+                    {
+                        "url": "Message",
+                        "valueString": "Information provided is for documents created from 2016-03-01 through 2016-03-29 inclusive."
+                    }
+                ]
+            }
+        },
+        {
+            "url": "http://cerner.hs.fhir.com/StructureDefinition/bundle-result-messages",
+            "valueExtension": {
+                "url": "http://cerner.hs.fhir.com/StructureDefinition/bundle-result-messages",
+                "extension": [
+                    {
+                        "url": "Severity",
+                        "valueString": "info"
+                    },
+                    {
+                        "url": "Code",
+                        "valueString": "fsp001"
```

Comment:
I would change 'Date parameter values passed with request were:”passed date input parameters”.' to 'Date parameter values passed with request were: <date parameter values>.' so it is more clear that the provided parameters are returned instead of a literal string.